front-end-side/src/Pages/ExpensesPages.js

```diff
@@ -30,10 +39,28 @@ const ExpensesPages = ({ categoryType }) => {
                         {
                             status ? (
                                 info.data.map((item) => {
+                                    const { date, seller, details, amount, category } = item;
+                                    if (category == categoryType)
+                                        return (
+                                            <tr>
+                                                <td>{date}</td>
+                                                <td>{seller}</td>
```

Comment:
Make same slice as in ReadData.js (in order if seller name or purpose would be too long)

Reply:
Okey, i'm gonna fix